website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -0,0 +1,275 @@
+   * A Custom amiFamily. You must ensure that the node you add the `karpenter.sh/unregistered` taint in your UserData.
+   * An Ubuntu AMI, as described earlier.
+
+### Downgrading
+// TODO 
+
+## Full Changelog
+* Features: 
+  * AMI Selector Terms has a new Alias field which can only be set by itself in `EC2NodeClass.Spec.AMISelectorTerms`
+  * Disruption Budgets by Reason in `NodePool.Spec.Disruption.Budgets`
+  * TerminationGracePeriod in the `NodePool.Spec.Template.Spec`.
+  * LOG_OUTPUT_PATHS and LOG_ERROR_OUTPUT_PATHS environment variables added
+* API Rename: NodePool’s ConsolidationPolicy WhenUnderutilized is now renamed to WhenEmptyOrUnderutilized
+* Behavior Changes: 
+  * Expiration now begins draining as soon as it’s expired. Karpenter does not wait for replacement capacity to be available before draining, but will start provisioning a replacement as soon as the node is expired and begins draining.
+  * Karpenter's generated NodeConfig now takes precedence when generating UserData with the AL2023 `amiFamily`. If you're setting any values managed by Karpenter in your AL2023 UserData, configure these through Karpenter natively (e.g. kubelet configuration fields). 
+  * Karpenter now adds a karpenter.sh/unregistered taint to nodes in injected UserData when using alias in AMISelectorTerms or non-Custom AMIFamily. When using AMIFamily: Custom, users will need to add this taint into their UserData, where Karpenter will automatically remove it when provisioning nodes. 
+* API Moves: 
+  * ExpireAfter has moved from the `NodePool.Spec.Disruption` block to `NodePool.Spec.Template.Spec`, and is now a drift-able field.
+  * KubeletConfig was moved to the EC2NodeClass from the NodePool.
+* RBAC changes: added `delete pods` | added `get, patch crds` | added `update nodes` | removed `create nodes`
+* Breaking API (Manual Migration Needed): 
+  * Ubuntu is dropped as a first class supported AMI Family
+  * `karpenter.sh/do-not-consolidate` (annotation), `karpenter.sh/do-not-evict` (annotation), and `karpenter.sh/managed-by` (tag) are all removed. `karpenter.sh/managed-by`, which currently stores the cluster name in its value, will be replaced by eks:eks-cluster-name
+  * The taint used to mark nodes for disruption and termination changed from `karpenter.sh/disruption=disrupting:NoSchedule` to `karpenter.sh/disrupted:NoSchedule`. It is not recommended to tolerate this taint, however, if you were tolerating it in your applications, you'll need to adjust your taints to reflect this.
+* Environment Variable Changes: 
+  * Environment Variable Changes
+  * LOGGING_CONFIG, ASSUME_ROLE_ARN, ASSUME_ROLE_DURATION Dropped
+  * LEADER_ELECT renamed to DISABLE_LEADER_ELECTION
+  * DRIFT_ENABLED dropped and promoted to Stable, cannot be disabled.
```

Comment:
```suggestion
  * `DRIFT=true` is no longer a `FEATURE_GATE` dropped and promoted to Stable, cannot be disabled.
```

There is no `DRIFT_ENABLED` environment variable -- only a `FEATURE_GATES` environment variable